content/applications/finance/fiscal_localizations/india.rst

```diff
@@ -166,6 +166,45 @@ Cancel`.
+
+Negative Lines Conversion in E-Invoice
+**************************************
+
+When dealing with E-invoice entries, there may be scenarios where negative lines are encountered. These negative lines are typically employed to represent discounts or adjustments associated with specific products HSN and tax rate or global discounts.
+
+Consider the following example:
+
++---------------------------------------------------------------------------------------------------+
+|                                     **Product Details**                                           |
++=======================+==============+==================+==============+==============+===========+
+| **Product Name**      | **HSN Code** | **Tax Excluded** | **Quantity** | **GST Rate** | **Total** |
++-----------------------+--------------+------------------+--------------+--------------+-----------+
+| Product A             | 123456       | 1,000            | 1            |    18%       |    1,180  |
++-----------------------+--------------+------------------+--------------+--------------+-----------+
+| Product B             | 239345       | 1,500            | 2            |    5%        |    3,150  |
++-----------------------+--------------+------------------+--------------+--------------+-----------+
+| Discount on Product A | 123456       | -100             | 1            |    18%       |    -118   |
++-----------------------+--------------+------------------+--------------+--------------+-----------+
+
+For clarity, negative lines are converted into positive discounts based on the HSN Code and GST rate. Here's the transformed representation:
+
++-------------------------------------------------------------------------------------------------------------+
+|                                         **Product Details**                                                 |
++==================+==============+==================+==============+==============+==============+===========+
+| **Product Name** | **HSN Code** | **Tax Excluded** | **Quantity** | **Discount** | **GST Rate** | **Total** |
++------------------+--------------+------------------+--------------+--------------+--------------+-----------+
+| Product A        | 123456       | 1,000            | 1            | 100          |  18%         |  1,062    |
++------------------+--------------+------------------+--------------+--------------+--------------+-----------+
+| Product B        | 239345       | 1,500            | 2            | 0            |  5%          | 3,150     |
++------------------+--------------+------------------+--------------+--------------+--------------+-----------+
+
+In this conversion, negative lines have been transformed into positive discounts, maintaining accurate calculations based on the HSN Code and GST rate. This ensures a more straightforward and standardized representation in the E-invoice records.
+
```

Comment:
If I understand correctly, this explanation is not specific to the example above so I suggest moving it to the intro (after "which means they need to be converted based on the HSN code and GST rate")

also : e-Invoice records

Reply:
It is related to the above example on how we manage the negative lines.